### ISSUE TITLE
Account creation screen option for proxy/Tor

### ISSUE THREAD

**Issue by licaon-kter** [Antox/Antox#324]
Either that (like ChatSecure) or provide access to settings (like Conversations)

**Comment by wiiaam** [comment]
So add access to settings in the create account menu? Would also allow users to change their language etc

**Comment by ghostlands** [comment]
@wiiaam yeah but it may be worth checking out #254 again as a prerequisite :/

**Comment by licaon-kter** [comment]
So I created another account for testing and looks like this can be achieved already, with a caveat.

After you name your account if you are not connected you can still go forward and enter the app and then you can access settings, proxy, whatever.

Now that's fine only if you have an actual firewall (and I recommend [NetGuard](https://github.com/M66B/NetGuard/) ;)) setup, else connections might get on the clearnet.

This trick does not void the need for an actual way to do this from the interface though.
